src/webxr/ARButton.ts

```diff
@@ -79,7 +88,8 @@ class ARButton {
 
       button.onclick = function () {
```

Comment:
Do you think we could use `addEventListener` instead?

Reply:
yep, I'll convert it the morning :smiley: